### PR TITLE
[WNMGDS-2249] Remove Mgov deprecated colors

### PR DESCRIPTION
**Pull request by line47**
## Summary

- Mgov had some semantic colors marked for deprecation and they have been removed
- Documentation has been updated 

## How to test

1. Run the doc site with `yarn start` and visit the migration page to see the additional non-semantic colors for Mgov that are being removed 

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.

### If this is a change to code:

- [ ] Created or updated unit tests to cover any new or modified code
- [ ] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [ ] Checked for spelling and grammatical errors

### CODE REVIEW

File: packages/docs/content/migration-guides/standardize-theme-colors.mdx [hunk 1]
```diff
@@ -322,6 +322,105 @@ Red is removed and was a non-semantic duplicate of `error`.
...
+    [{ name: '--red-500',   value: '#b20000' }, null],
+    [{ name: '--red-700',  value: '#a00000' }, null],
+  ]}
+/>
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+### Teal
```
Comment (line47) [inline packages/docs/content/migration-guides/standardize-theme-colors.mdx:340]: @pwolfert and @forrestbaer not sure of the best way to format this Mgov specific content right now these headings like `Teal` show up on on table of contents for every theme though only applicable to Mgov :-/ 

Idea?

Comment (pwolfert) [reply packages/docs/content/migration-guides/standardize-theme-colors.mdx:340]: I don't have a fix for it, but a workaround could be to drop them under another heading so all the individual colors don't show up. I'm trying to think of where to put things to achieve that.

I saw you put red under red, so in that same line of thinking,
- Teal could go under primary
- Blue could go under secondary
- Yellow could go under gold
- Black could go under neutral

Comment (line47) [reply packages/docs/content/migration-guides/standardize-theme-colors.mdx]: Thanks for the suggestion @pwolfert this is ready for 👀

**Review by pwolfert — APPROVED** [review]
Looks great!

**Pull request merged**